waterlog/src/Stylesheets/_alerts.scss

```diff
@@ -0,0 +1,47 @@
+table {
+  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
+  border-collapse: collapse;
+  color: white;
+  width: 100%;
+
+}
+
+tr {
+  border-bottom: 1px solid white;
+  width: 100%;
+}
+
+// tr:hover {
```

Comment:
Add a TODO if you intend to use it later on

Reply:
Deleted comments